schema/dereferenced-release-schema.json

```
@@ -2173,6 +2173,29 @@
                       "string",
                       "null"
                     ]
+                  },
+                  "weight": {
+                    "title": "Weight",
+                    "description": "The weight of the item unit.",
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "title": "Name",
+                        "description": "The name of the weight, using UN/CEFACT Common Codes for Units of Measurement.",
+                        "type": [
+                          "string",
+                          "null"
+                        ]
+                      },
```

This differs from the proposal in https://github.com/open-contracting/standard/issues/1110#issuecomment-730064497
```suggestion
                      "unit": {
                        "title": "Unit",
                        "description": "The code for the unit in which the weight is specified, using UN/CEFACT Common Codes for Units of Measurement.",
                        "type": [
                          "string",
                          "null"
                        ]
                      },
```